scratches/test_of_qr.py

```diff
@@ -0,0 +1,94 @@
+import telebot  # importing Telegram API classes and functions
```

Comment:
General comment. I'm a bit confused with this file. The name implies that this is the test. But business logic is here. I don't think we should include scratch into pull requests. If you want to show code, it is better to publish then in a separate branch.